src/components/structure/interacting/navigation/navigation.tsx

```diff
@@ -17,10 +17,16 @@ type Translations = {
 const Navigation = () => {
```

Comment:
Hier kann dann weiter oben in den Imports, zusätzlich der `UISidebarType`-Enum importiert werden:

```ts
import StoreContext, { GlobalSearchEntityType, UISidebarType } from '../../../../store/StoreContext';
```